model/client4.go

```diff
@@ -1226,6 +1226,17 @@ func (c *Client4) DeleteUser(userId string) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// PermanentDeleteUser deletes a user in the system based on the provided user id string.
+// If the EnableAPIUserDeletion config option is not set this falls back to user deactivation.
```

Comment:
nit: the comment needs to be updated with the new behaviour

Reply:
Good catch! fixed.